### ISSUE TITLE
cannot use the block ui in ngOnInit()

### ISSUE THREAD

**Issue by mobh13** [kuuurt13/ng-block-ui#119]
I need to block my UI while I'm loading the data using sync but it seems that it is not working, I can confirm that when I use any sync code it dose not work.

```
ngOnInit(): void {
    this.form = new FormGroup({
      accountT: new FormControl(null, Validators.required),
      accountValue: new FormControl(null, Validators.required)
    });
    this.userObjectId = Backendless.LocalCache.get("current-user-id");
    if (
      this.userObjectId != null &&
      Backendless.UserService.isValidLoginSync()
    ) {
      const contact = Backendless.Data.of("Users").findByIdSync({
        objectId: this.userObjectId
      });
      if (contact != null) {
        this.user = contact;
        this.username = contact["username"];
        this.name = contact["name"];
        this.views = contact["views"];
        this.avatar = contact["avatar"];
        const queryBuilder = new Backendless.DataQueryBuilder();
        queryBuilder.setWhereClause("UserID = '" + this.userObjectId + "'");
        const acc = Backendless.Data.of("Accounts").findSync(queryBuilder);
        if (acc != null) {
          acc.forEach(function(element) {
            const account = new Account();
            account.id = element["objectId"];
            account.value = element["Value"];
            const queryBuilder2 = new Backendless.DataQueryBuilder();
            queryBuilder.setWhereClause(
              "Id = '" + element["AccountType"] + "'"
            );
            const type = Backendless.Data.of("AccountsType").findSync(
              queryBuilder
            );
            account.icon = type[0]["Icon"];
            account.type = element["AccountType"];
            this.accounts.push(account);
            console.log(element["Clicks"]);
            console.log(type[0]["Icon"]);

            this.clicks = this.clicks + element["Clicks"];
            // this.accounts.push(account);
          }, this);
        }
        const types = Backendless.Data.of("AccountsType").findSync();
        types.forEach(function(element) {
          this.accountsType.push(
            new AccountsType(element["Icon"], element["Id"], element["Name"])
          );
        }, this);
      } else {
        this.router.navigateByUrl("/login");
      }
    }
    this.blockUI.stop();
  }
```

**Comment by kuuurt13** [comment]
@mobh13 I don't ever see you call `this.blockUI.start()` in your `ngOnInit()` method.

**Comment by mobh13** [comment]
it seems that I deleted it in this snippet but I added it and it is the same

**Comment by kuuurt13** [comment]
@mobh13 this looks like it might be related to #120. Can you please try updating to `3.0.0-beta.5` and let me know if it fixes your error (`npm i -S ng-block-ui@next`)

**Comment by kuuurt13** [comment]
Fixed by #113
